python/basic/advanced_img_io.ipynb

```diff
@@ -4,6 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "### Download the whole repository with command:\n",
```

Comment:
Why should all notebooks have this dependency? We should add a section something like below only for notebooks that has some dependency on file:

### Prerequisite
1. This notebook requires this file [link to file]. Make sure to download this file along with the notebook.